app/actionUpdate/get.controller.test.js

```diff
@@ -96,6 +102,7 @@ describe('getActionUpdate', () => {
         })
       )
     })
+    it('should add a timeline entry if created and createdBy are defined', () => {})
```

Comment:
Missing test?

Reply:
👍 I've put the test in, but it was sort of being tested anyway and will probably be redundant once the new version of the API is released.